src/app/home/problem-list/problem.model.ts

```diff
@@ -0,0 +1,5 @@
+export interface Problem {
```

Comment:
Suggestion: Use the convention `IProblem` ('I' prefix for Interface - easy code readability).

Reply:
Updated
[Commit link](https://github.com/armory-uci/vital/pull/8/commits/066d6038cd7ea68ebd6bcc242bd6fed17dc374fb)